example.env

```diff
@@ -64,9 +64,6 @@ MINIMUM_DURATION=2
 # Python pytz timezone
 LOCAL_TIME_ZONE=US/Eastern
 
-# The Lambda Function alias that points to the "live" version of the function
```

Comment:
I've never once had a reason for this to be configurable so I just made it a constant in `names.py`